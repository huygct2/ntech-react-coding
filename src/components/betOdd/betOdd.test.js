import React from 'react';
import renderer from 'react-test-renderer';
import BetOdd from './betOdd';

test('Link changes the class when hovered', () => {
  const component = renderer.create(
    <BetOdd returnWin={1.1} />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
