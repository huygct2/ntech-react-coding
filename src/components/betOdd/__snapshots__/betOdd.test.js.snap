// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BetOdd component 1`] = `
<div
  className="bet-odd box"
>
  <span>
    1.1
  </span>
</div>
`;
